docs/Installation.md

```diff
@@ -731,63 +700,71 @@ service postgresql initdb
 service postgresql start
 ```
 
-Initialize Zonemaster database and user (unless you keep an old database):
+To create the database and the database user (unless you keep an old database)
+run the command on one line.
 
 ```sh
-sudo -u postgres createuser -P zonemaster
+psql -U postgres -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';"  -c "CREATE DATABASE zonemaster WITH ENCODING 'UTF8';"
 ```
 
-This will ask for a password that should be copied in `backend_config.ini`.
+Update the `/etc/zonemaster/backend_config.ini` file with username and password
+if non-default values are used.
 
-```sh
-sudo -u postgres createdb -O zonemaster -E UTF8 zonemaster
-```
+Now go back to "[Database configuration](#53-database-configuration-freebsd)"
+to create the database tables and then continue with the steps after that.
 
 
-### C. Cleaning up the database
+## 10. Cleaning up the database
 
 If, at some point, you want to delete all traces of Zonemaster in the database,
 you can run the file `cleanup-mysql.sql` or file `cleanup-postgres.sql`
 as a database administrator. Commands
 for locating and running the file are below. It removes the user and drops the
 database (obviously taking all data with it).
 
-#### C.1. MySQL
+### 10.1. MariaDB and MySQL
 
 ```sh
-cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
-mysql --user=root --password < ./cleanup-mysql.sql
+mysql --user=root -p < `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`/cleanup-mysql.sql
 ```
 
-#### C.2. PostgreSQL
+### 10.2. PostgreSQL
+
+CentOS, Debian and Ubuntu:
 
 ```sh
-cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
-sudo -u postgres psql -f ./cleanup-postgres.sql # MUST BE VERIFIED!
+sudo -u postgres psql -f $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/cleanup-postgres.sql # MUST BE VERIFIED!

```

Comment:
I think we can safely remove the comment here.

Reply:
Removed.